Tests/Fixtures/TestProject/Project.xcodeproj/project.pbxproj

```diff
@@ -507,6 +510,7 @@
 			files = (
 				535A98A3E3B74E09891D977F /* TestFramework.framework in Embed Frameworks */,
 				BD1419893577E6CEDF8CBA83 /* Result.framework in Embed Frameworks */,
+				BB06A57E259D0D2A001EA21F /* Result.framework in Embed Frameworks */,
```

Comment:
As you can see, new copy of Result dependency is added. This is result of fixing Platforms equatable. I tried to understand this behaviour, but I failed. I do not even know if is error or not.